packages/vanilla/src/sass/components/button/_base.scss

```diff
@@ -12,44 +12,37 @@
   min-height: 2.25rem;
 
   & > i {
+    font-size: 1rem;
```

Comment:
Looks like this is increasing the height of the button to 37px, I think this can be removed since buttons get their font size from the overall default font size.

Reply:
I'm going to just comment this out as a reminder then. Jeremy is specing these out as individual SVGs rather than fonts now, so the plan was to size them according to spec and update the selector when we switch to SVGs.